simulator/src/main/java/com/hazelcast/simulator/utils/PropertyBindingSupport.java

```diff
@@ -178,6 +164,19 @@ private static Field findPropertyField(Class clazz, String property) {
         }
     }
 
+    private static boolean setValue(Object object, String value, Field field) throws IllegalAccessException {
```

Comment:
Can we blame the 36°C in Istanbul for that? I moved that check out of the method.

Good catch!